### DISCUSSION TITLE
[Select] 能自定义options 的每个option的样式或者slot吗？

### DISCUSSION THREAD

**Discussion by RSS1102** [ant-design/ant-design#42327] [Q&A] [answered]
我想根据Select的`options`数据来自定义每个`option`，比如给每个`option`不同的文字颜色或者给不同的图标。

**Comment by kiner-tang** [comment, marked as answer]
使用 JSX 的方式应该可以满足你的需求：
<img width="653" alt="image" src="https://github.com/ant-design/ant-design/assets/10286961/23d01791-bc93-425f-898e-a80bcf99a2de">
<img width="547" alt="image" src="https://github.com/ant-design/ant-design/assets/10286961/8222452e-d1c4-42ca-bd31-b066518fdfba">

https://codesandbox.io/s/8tzq4q

**Reply by RSS1102** [reply]
谢谢
